src/main/resources/interface-specification.yml

```
@@ -1401,6 +1401,86 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /producer/eservices:
```

not sure if can help, but I was planning to have an endpoint `/producers` for the list of tenants that are producers (duh).
Could it make sense to rename this to 
```suggestion
  /producers/eservices:
```
?